tensorflow/core/kernels/slice_op.cc

```diff
@@ -205,6 +205,207 @@ class SliceOp : public OpKernel {
   }
 };
 
+#ifdef INTEL_MKL
```

Comment:
MKL Slice is not in a separate file because it is not optimized by MKL API yet.
Current optimization is to speed up Eigen version using OpenMP.

Another reason for not using a separate file is build rules for Slice op. They are
currently written in such a way (using slice* prefix) that adding mkl_slice would
require sizable changes to the build file.